OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -49,28 +51,33 @@ std::string LiveFunctionsDataView::GetValue(int a_Row, int a_Column) {
     return "";
   }
 
-  Function& function = GetFunction(a_Row);
-  const std::shared_ptr<FunctionStats> stats = function.stats();
+  protos::FunctionInfo& function = GetFunction(a_Row);
+  const protos::FunctionStats& stats = function.stats();
 
   switch (a_Column) {
     case COLUMN_SELECTED:
       return FunctionUtils::IsSelected(function) ? "X" : "-";
     case COLUMN_NAME:
       return FunctionUtils::GetDisplayName(function);
     case COLUMN_COUNT:
-      return absl::StrFormat("%lu", stats->m_Count);
+      return absl::StrFormat("%lu", stats.count());
     case COLUMN_TIME_TOTAL:
-      return GetPrettyTime(stats->m_TotalTimeMs);
+      return GetPrettyTime(
+          absl::ToDoubleMilliseconds(absl::Nanoseconds(stats.total_time_ns())));
     case COLUMN_TIME_AVG:
-      return GetPrettyTime(stats->m_AverageTimeMs);
+      return GetPrettyTime(absl::ToDoubleMilliseconds(
+          absl::Nanoseconds(stats.average_time_ns())));
     case COLUMN_TIME_MIN:
-      return GetPrettyTime(stats->m_MinMs);
+      return GetPrettyTime(
+          absl::ToDoubleMilliseconds(absl::Nanoseconds(stats.min_ns())));
     case COLUMN_TIME_MAX:
-      return GetPrettyTime(stats->m_MaxMs);
+      return GetPrettyTime(
+          absl::ToDoubleMilliseconds(absl::Nanoseconds(stats.max_ns())));
```

Comment:
I am aware of @dpallotti comment from earlier. This here seems like an unnecessary amount of code duplication. I would propose to add a second GetPrettyTime to Utils.h
`std::string GetPrettyTime(uint64_t nanoseconds) { ... }`

Reply:
Ideally, I wanted to replace `std::string GetPrettyTime(double milliseconds)` with `std::string GetPrettyTime(uint64_t nanoseconds)` in the next PR to reduce amount  of changes. But probably I can do it in two steps (just add a second function for now and then get rid of the first one).

Reply:
Please no overloads for two numeric types with different units. It's bad enough to convey the unit in the name of the parameter. GetPrettyTime should take a absl::Duration.

Reply:
I like the idea to use absl::Duration, and it doesn't require too much changes. I added a commit with this change (469cf9900af0df4f43f87b3dbcbd7ad52765f700).